src/internal/rest.rs

```diff
@@ -8,21 +8,19 @@ use bytes::Bytes;
 use chrono::{DateTime, Utc};
 use percent_encoding::SIMPLE_ENCODE_SET;
 use reqwest::{
-    header::HeaderMap,
-    r#async::{Client as RClient, Request as ARequest},
+    header::{HeaderMap, HeaderValue, CONTENT_TYPE},
+    r#async::{Client as RClient, Request as ARequest, RequestBuilder},
```

Comment:
did you figure out that the async namespace is still the preferred way to import here? 
https://github.com/seanmonstar/reqwest/blob/80ba8cc15066c2a6d35d68b058e795be5e01cd94/src/lib.rs#L301 makes me think otherwise.

Reply:
No, I just pushed the fix.